src/Header.jsx

```diff
@@ -1,14 +1,29 @@
+import Dropdown from "react-bootstrap/Dropdown";
+import GetTopics from "./GetTopics";
+import { useState } from "react";
 export default function Header() {
+  const [topics, setTopics] = useState([]);
+  const [topicChoice, setTopicChoice] = useState("");
+
   return (
     <>
       <div key="nav-bar" className="nav-bar">
         <div key="flex-box-left-holder" className="flex-box-left-holder">
-          <h2 key="nav-topics" className="nav-topics">
-            Topics
-          </h2>
-          <h2 key="nav-articles" className="nav-articles">
-            Articles
-          </h2>
+          <div className="dropdown">
+            <button className="dropbtn">
+              <h2>Topics</h2>
+            </button>
+            <div id="myDropdown" className="dropdown-content">
```

Comment:
I think this dropdown looks great in the header where it is, however, at the moment, when we click a single article it remains in the header but is redundant as the user cannot use it at that point. Consider having the dropdown only render when it can actually be used 👍